thirdparty/poppler/qt5/poppler-optcontent.cc

```diff
@@ -427,4 +427,4 @@ namespace Poppler
   }
 }
 
-#include "poppler-optcontent.moc"
+//#include "poppler-optcontent.moc"
```

Comment:
Well, yes, but my question was more as to why does it fail for @ 
aryamanvinchhi and how to get it to work?
Also that file seems to be empty, at least here on my machine